pkg/apis/sources/v1/container_lifecycle_test.go

```diff
@@ -133,6 +145,7 @@ func TestContainerSourceStatusIsReady(t *testing.T) {
 			s.InitializeConditions()
 			s.PropagateSinkBindingStatus(&readySinkBinding.Status)
 			s.PropagateReceiveAdapterStatus(unknownDeployment)
+			s.MarkOIDCIdentityCreatedUnknown("unknown", "")
```

Comment:
Same as above: should use `s.MarkOIDCIdentityCreatedSucceeded()`, as we want to check the status of the deployment